cmd/start/main.go

```diff
@@ -49,6 +49,42 @@ func main() {
 
 	svisor := supervisor.New("flypg", 5*time.Minute)
 
+	if timeout, exists := os.LookupEnv("FLY_SCALE_TO_ZERO"); exists {
+		sql := "select count(*) from pg_stat_activity where usename != 'repmgr' and usename != 'flypgadmin' and backend_type = 'client backend';"
+		duration, err := time.ParseDuration(timeout)
+		fmt.Println("Configured scale to zero")
+		if err != nil {
+			fmt.Printf("failed to parse FLY_SCALE_TO_ZERO duration %s", err)
+		} else {
+			go func() {
+				timeout := time.After(duration)
+				for {
+					select {
+					case <-timeout:
+						localConn, err := node.NewLocalConnection(ctx, "postgres", node.OperatorCredentials)
+						if err != nil {
+							svisor.Stop()
```

Comment:
It seems like there's potential for this to race with PG coming up on boot?  Also, we may not want to shut down everything in case this error is intermittent.